src/MemoryTracing/MemoryTracingUtils.cpp

```diff
@@ -214,12 +214,12 @@ ErrorMessageOr<int64_t> ExtractRssAnonFromProcessStatus(std::string_view status_
   return ErrorMessage("RssAnon value not found in the file content.");
 }
 
-ErrorMessageOr<ProcessMemoryUsage> GetProcessMemoryUsage(int32_t pid) noexcept {
+ErrorMessageOr<ProcessMemoryUsage> GetProcessMemoryUsage(uint32_t pid) noexcept {
```

Comment:
Isn't this linux specific API - in which case it should probably use native pid types.

Reply:
Done, thanks.